### PR TITLE
merged host and path configuration values into address and set default to production

### PR DESCRIPTION
**Pull request by avalcepina**

### CODE REVIEW

File: cmd/adapters/adapters.go [hunk 1]
```diff
@@ -60,15 +61,20 @@ func (c *customTransport) RoundTrip(r *http.Request) (*http.Response, error) {
 	return resp, nil
 }
 
-func NewApiClient(host, basePath, apiVersion, apikey string) *client.Anansi {
+func NewApiClient(address, apiVersion, apikey string) (*client.Anansi, error) {
 
-	transport := httptransport.New(host, basePath, []string{"http"})
+	parsedUrl, err := url.Parse(address)
+	if err != nil {
+		return nil, err
```
Comment (TymoteuszGach) [inline cmd/adapters/adapters.go:68]: can you add context to the error?

File: cmd/attachservice.go [hunk 2]
```diff
@@ -33,7 +33,10 @@ var attachServiceCmd = &cobra.Command{
 
 		logging.Info("Attaching service", logging.NewPair("service-name", serviceName))
 
-		httpClient := adapters.NewApiClient(Config.VampCloudHost, Config.VampCloudBasePath, ApiVersion, Config.APIKey)
+		httpClient, err := adapters.NewApiClient(Config.VampCloudAddr, ApiVersion, Config.APIKey)
+		if err != nil {
+			return err
```
Comment (TymoteuszGach) [inline cmd/attachservice.go:38]: Should we add context to the error here as well? (there are more places in other files too)

Comment (avalcepina) [reply cmd/attachservice.go:38]: I think if we add it in the adapter there's no need to add it here as well. After adding to the adapter the error will already state that creating the api client failed

Comment (TymoteuszGach) [reply cmd/attachservice.go:38]: ok

**Review by TymoteuszGach — APPROVED** [review]

File: cmd/attachservice.go (hunk 2 above)
Comment (avalcepina) [reply cmd/attachservice.go:38]: I added some errors wrapping where it looked like it would be useful

**Review by TymoteuszGach — APPROVED** [review]
looks better now with those wrapped errors 👍

**Pull request merged**